packages/components/src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -1,24 +1,29 @@
-import { Component, h, Prop } from '@stencil/core';
+import { Component, h, Element } from '@stencil/core';
 
 @Component({
   tag: 'ifx-breadcrumb',
   styleUrl: 'breadcrumb.scss',
   shadow: true,
 })
 export class Breadcrumb {
-  @Prop() breadcrumbs: { text: string, url: string }[];
+  @Element() el;
+
+  componentDidLoad() { 
+    const breadcrumbItems = this.el.querySelectorAll('ifx-breadcrumb-item')
+    for(let i = 1; i < breadcrumbItems.length; i++) { 
+      let label = breadcrumbItems[i].querySelector('ifx-breadcrumb-item-label')
+      let container = label.shadowRoot.querySelector('.breadcrumb-item-label-container')
+      if(container) { 
+        container.classList.add('margin')
+      }
+    }
+  }
 
   render() {
     return (
       <nav aria-label="breadcrumb">
```

Comment:
Good work on including the aria-label!
However: A11y Improvement: In your code, the aria-label="breadcrumb" should have a more descriptive label. For example, "Page navigation breadcrumb" would be more informative to users who rely on screen readers.